com.unity.render-pipelines.high-definition/Documentation~/whats-new-10-0.md

```diff
@@ -112,6 +112,11 @@ For more information, see the Lighting panel section in the [HDRP debug window](
 
 The Render Pipeline Debug window now has a new Volume panel which you can use to visualize the Volume components that affect a specific Camera. For each Volume that contributes to the final interpolated value, this panel shows the value of each property and whether or not it is overridden. It also calculates the Volume's influence percentage using the Volume's weight and blend distance. For more information, see the Volume panel section in the [HDRP debug window](Render-Pipeline-Debug-Window.md#VolumePanel).
 
+#### Quad Overdraw and Vertex Density
+
+Quad Overdraw highlights GPU quads running multiple fragment shaders, which is mainly caused by small or thin triangles while Vertex Density displays pixels running multiple vertex shaders.
```

Comment:
please add that it is not supported on metal

Reply:
it's already there in the line under